CONTRIBUTING.md

```diff
@@ -210,14 +210,6 @@ make run PROJECT_PATH=<path-to-your-test-project>/new-kedro-project
 
 > **Note**: Once the backend development server is launched at port 4142, the local app will always pull data from that server. To prevent this, you can comment out the proxy setting in `package.json` and restart the dev server at port 4141.
 
-#### Launch the development server with the `SQLiteSessionStore`
-
-Kedro-Viz provides a `SQLiteSessionStore` that users can use in their project to enable experiment tracking functionality. If you want to use this session store with the development server, make sure you don't use a relative path when specifying the store's location in `settings.py`. For example, `demo-project` specifies the local `data` directory within a project as the session store's location as follows:
-
-```python
-from kedro_viz.integrations.kedro.sqlite_store import SQLiteStore
-SESSION_STORE_ARGS = {"path": str(Path(__file__).parents[2] / "data")}
-```
```

Comment:
We need to make changes to all the starter templates (may be a note saying kedro-viz 10.1.0 does not support ET) as a follow up - https://github.com/kedro-org/kedro-starters/blob/main/spaceflights-pandas-viz/%7B%7B%20cookiecutter.repo_name%20%7D%7D/src/%7B%7B%20cookiecutter.python_package%20%7D%7D/settings.py

Reply:
hey, @merelcht has already raised this concern, and I’ll create a ticket for it. For now, we’ve added [a deprecation warning for Kedro datasets](https://github.com/kedro-org/kedro-plugins/issues/954) to alert users about this change coming in the next release.